dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -179,6 +180,9 @@ private constructor(
          * @param enabled true by default
          */
         fun setRumEnabled(enabled: Boolean): Builder {
+            if (rumConfig.applicationId == UUID(0, 0)) {
+                sdkLogger.e(Datadog.MESSAGE_NOT_INITIALIZED_WITH_APP_ID)
```

Comment:
This should only be a warning. Their data won't be lost, it'll still be visible in the explorer, it just wont be properly categorized and linked with the appropriate application ID